cogboard-app/src/main/kotlin/com/cognifide/cogboard/widget/BaseWidget.kt

```diff
@@ -106,6 +106,14 @@ abstract class BaseWidget(
         return config
     }
 
+    protected fun updateStateByCopingPropsToContent(props: Set<String>) {
+        val content = JsonObject()
+        props.forEach {
```

Comment:
`props.map`?

Reply:
`forEach` will serve better in this case